openlibrary/plugins/upstream/borrow.py

```diff
@@ -148,7 +148,18 @@ def POST(self, key):
                 redirect_url += '&_autoReadAloud=' + i._autoReadAloud
             raise web.seeother(redirect_url)
 
-        if action == 'return':
+        if action == "request":
+            try:
+                return requests.post(
+                    "https://hooks.slack.com/services/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",
```

Comment:
This test key needs to come from config
```suggestion
                    "REDACTED",
```